train:
  ros__parameters:
    environment: 'CarBeat' # CarGoal, CarWall, CarBlock, CarTrack, CarBeat
    track: 'multi_track' # track_1, track_2, track_3, multi_track, multi_track_testing -> only applies for CarTrack
    car_name: 'f1tenth'
    algorithm: 'TD3'
    g: 1
    buffer_size: 50000
    observation_mode: 'lidar_only'
